sdk/managed/test/Microsoft.WindowsAzure.MobileServices.Android.Test/LoginActivity.cs

```diff
@@ -16,16 +16,10 @@ public class LoginActivity : Activity
         static class Keys
         {
             public const string MobileServiceUri = "MobileServiceUri";
-            public const string MobileServiceKey = "MobileServiceKey";
             public const string TagExpression = "TagExpression";
 
             public const string AutoStart = "AutoStart";
-            public const string MasterRunId = "MasterRunId";
             public const string RuntimeVersion = "RuntimeVersion";
-            public const string CliendId = "CliendId";
-            public const string ClientSecret = "ClientSecret";
-            public const string DayLightUrl = "DayLightUrl";

```

Comment:
Xamarin solution is missing changes for daylight. Will submit another PR to master branch